src/main/scripts/org/reaktivity/specification/nukleus/tcp/control/route.ext/server/controller.rpt

```diff
@@ -20,6 +20,8 @@ property nukleiRG010001C ${nuklei:directory("target/nukleus-itests").controlCapa
 
 property controlRG010001C ${nukleiRG010001C.control("tcp")}
 
+property routeAuthorization [0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00]
```

Comment:
Should this value be `0x0000_000000000000L` ?

Reply:
yes, fixed